src/push/jira.js

```diff
@@ -0,0 +1,43 @@
+const config = require("../config");
+const notifyConfig  = config.get("notify");
+const schnackEvents = require('../events');
+
+if (notifyConfig.jira) {
+
+    var JiraClient = require('jira-connector');
+    var jira = new JiraClient({
+        host: notifyConfig.jira.host,
+        basic_auth: {
+            base64: notifyConfig.jira.basic_auth.base64
+        }
+    });
+
+    schnackEvents.on('new-comment', (event) => {
```

Comment:
This is an ES6 function format, but I can't see where this would be compiled down to. So either this doesn't run, or I have missed something.

If this does work, I'd recommend using the same syntax everywhere possible for consistency.

Reply:
This is consistent with the formatting in other `push` notifiers, @DesignyourCode (take a look at the `slack.js` and `sendmail.js` files in the `/push` directory, for reference).

I'd be inclined to leave this one as-is